java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java

```diff
@@ -56,7 +57,9 @@ public static void main(String[] args) {
                     testClientSetGet(LettuceAsyncClient::new, runConfiguration, true);
                     break;
                 case GLIDE:
+                    // run testClientSetGet on GLIDE async client
```

Comment:
I feel like this comment is redundant

Reply:
Probably.  Consistent with the comments above, but all of them are really gratuitous.